src/test/java/seedu/resireg/logic/LogicManagerTest.java

```diff
@@ -162,9 +164,16 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     private void assertHistoryCorrect(String ...expectedCommands) {
         try {
             CommandResult result = logic.execute(HistoryCommand.COMMAND_WORD);
+
+            List<String> expected = Arrays.asList(expectedCommands);
```

Comment:
Good to refactor this to `getResultAsString()` as well.

Reply:
Resolved. Again, I've extracted the above code fragment into a method called getExpectedResultString() to adhere to SLAP.